src/map/mob.c

```diff
@@ -1413,6 +1413,7 @@ int mob_randomwalk(struct mob_data *md, int64 tick) {
 	nullpo_ret(md);
 
 	if(DIFF_TICK(md->next_walktime,tick)>0 ||
+		status_get_mode(&md->bl)&MD_NORANDOM_WALK ||
```

Comment:
formatting look like broken.
in other lines probably used tab + spaces, and in added line tabs only.
also ``&`` without spaces

Reply:
OK this need serious discussion ......

1. see **Chapter 3: Placing Braces and Spaces**
ok this one I was wrong, I have to use one space around the binary operators
probably also need a bracket to enclose them
>Use one space around (on each side of) most binary and ternary operators, such as any of these:

	=  +  -  <  >  *  /  %  |  &  ^  <=  >=  ==  !=  ?  :

2. see **Chapter 2: Breaking long lines and strings**
there is no example given on how to properly break a line
any example ?

Reply:
if you dont know how to do, do like in near lines.
inside if condition it using tab for offset to ``if`` location and spaces for offset from ``if``
but it your code look like was used only tabs.

Use same way like below lines after your code